atom/browser/atom_browser_context.cc

```diff
@@ -128,6 +146,11 @@ AtomBrowserContext::~AtomBrowserContext() {
   // Notify any keyed services of browser context destruction.
   BrowserContextDependencyManager::GetInstance()->DestroyBrowserContextServices(
       this);
+
+#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
+  // TODO: Why?
+  extension_system_ = nullptr;
```

Comment:
This originates from [the extension shell code](https://cs.chromium.org/chromium/src/extensions/shell/browser/shell_browser_main_parts.cc?sq=package:chromium&dr&g=0&l=291). It doesn't seem useful from a functional benefit, but maybe rather signifies to a reader that it should no longer be used within that context.